docs/DEVELOPER_GUIDE.md

```diff
@@ -1,349 +1,4 @@
-# Developer Guide 
+# Developer Guide
 
-This guide is intended for plugin developers. If you are not developing kubectl
-plugins, read the [User Guide](./USER_GUIDE.md) to learn how to use krew.
-
-This guide explains how to package, test, run plugins locally and make them
-available on the krew index.
-
-<!-- TOC depthFrom:2 -->
-
-- [Developing a Plugin](#developing-a-plugin)
-    - [Packaging plugins for krew](#packaging-plugins-for-krew)
-    - [Writing a plugin manifest](#writing-a-plugin-manifest)
-        - [Specifying platform-specific instructions](#specifying-platform-specific-instructions)
-        - [Specifying files to install](#specifying-files-to-install)
-        - [Specifying plugin executable](#specifying-plugin-executable)
-        - [Specifying a plugin download URL](#specifying-a-plugin-download-url)
-- [Installing Plugins Locally](#installing-plugins-locally)
-- [Publishing Plugins](#publishing-plugins)
-    - [Submitting a plugin to krew](#submitting-a-plugin-to-krew)
-    - [Updating existing plugins](#updating-existing-plugins)
-
-<!-- /TOC -->
-
-Find Krew maintainers on [#krew](https://kubernetes.slack.com/messages/krew)
-room on Kubernetes Slack channel to ask questions, or simply open an issue if
-something is missing.
-
-## Developing a Plugin
-
-Before creating a plugin, read the [Kubernetes Plugins documentation][plugins].
-
-**If you are writing a plugin in Go:** Consider using the
-[cli-runtime](https://github.com/kubernetes/cli-runtime/) project which is
-designed to provide the same command-line arguments, kubeconfig parser,
-Kubernetes API REST client, and printing logic. Look at [sample-cli-plugin](https://github.com/kubernetes/sample-cli-plugin)
-for an example of a kubectl plugin.
-
-**Unofficial Quickstart for Go:** For Go plugins, there's a
-[GitHub template repo](https://github.com/replicatedhq/krew-plugin-template)
-that implements these best practices and also configures GoReleaser and a
-GitHub Action to create releases when a tag is pushed. Note: this template
-repo is not maintained by the Krew project.
-
-Below you will create a small plugin named `foo` which prints the environment
-variables to the screen and exits.
-
-Read the [Naming Guide](./NAMING_GUIDE.md) for choosing a name for your plugin.
-
-Create an executable file named `kubectl-foo` with the following contents:
-
-```bash
-#!/usr/bin/env bash
-env
-```
-
-Then make this script executable:
-
-    chmod +x ./kubectl-foo
-
-(Since this plugin requires `bash`, it will only work on Unix platforms. If you
-need to support windows, develop a `kubectl-foo.exe` executable.)
-
-Now, if you place this plugin to anywhere in your `$PATH` directories, you
-should be able to call it like:
-
-    kubectl foo
```

Comment:
This section is gone from our new docs. I think it's worth adding a new page with something like this.

Reply:
This is at http://localhost:1313/docs/developer-guide/develop/plugin-development/

Reply:
Ah right. O_O